src/rovers/roversRouter.ts

```diff
@@ -0,0 +1,14 @@
+import express from "express";
+import axios from "axios";
+
+const apiKey = 'api_key=6xr9QIaOXNnYO0KLwLTfgUZSdH2gpuni7VOTAZYt';
+const apiUrlNASA = 'https://api.nasa.gov/mars-photos/api/v1';
```

Comment:
Not sure how to nicely have a const file, just use exports?

Reply:
Yes, that should work, good idea!